### PR TITLE
API-1020 - Allow an array for the `apisSorter` swaggerOptions schema

### PR DESCRIPTION
**Pull request by jonesdhtx**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const optionsSchema = Joi.object({
     validatorUrl: Joi.string().allow(null, false).optional(),
     docExpansion: Joi.string().allow(null).valid(['none', 'list', 'full']).optional(),
     defaultModelRendering: Joi.string().allow(null).valid(['model', 'schema']).optional(),
-    apisSorter: Joi.string().allow(null).valid(['none', 'alpha']).optional(),
+    apisSorter: Joi.alternatives().try(Joi.array().items(Joi.string()), Joi.string().allow(null).valid(['none', 'alpha'])).optional(),
```
Comment (zack37) [inline lib/index.js:20]: Have you looked at `Joi.array().single()`? Might make this a little cleaner

Comment (jonesdhtx) [inline lib/index.js:20]: Ah cool - thanks for pointing that out, good to know.

For this use case, the string validation is different depending on whether or not it's in the array.  If not an array it is an enum, however if an array it's a set of tags which can be any non-empty string

Comment (zack37) [inline lib/index.js:20]: Ah. Ok. Fair enough

**Comment by zack37** [comment]
👍

**Pull request merged**